### PR TITLE
Adding configuration to build automation using github actions

### PR DESCRIPTION
**Pull request by Hari-1027**
Adding GitHub Actions Workflow to,

- Install the dependencies
- Build the service

### CODE REVIEW

**Comment by Hari-1027** [comment]
@HarishGangula Kindly verify the PR

File: aml.cicd.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        uses: docker/login-action@v3
+        with:
+          username: ${{ vars.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Build docker image and push
+        uses: docker/build-push-action@v5
+        with:
+          context: ./command-service
```
Comment (HarishGangula) [inline aml.cicd.yml:43]: copy paste issue

File: aml.cicd.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          platforms: linux/amd64
+          push: true
+          tags: ${{ vars.DOCKERHUB_USERNAME }}/flink-command-service:${{ github.ref_name }}
```
Comment (HarishGangula) [inline aml.cicd.yml:46]: copy paste issue

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: aml.cicd.yml (hunk 1 above)
Comment (Hari-1027) [reply aml.cicd.yml]: removed the command-service

File: aml.cicd.yml (hunk 2 above)
Comment (Hari-1027) [reply aml.cicd.yml]: removed the command-service

**Review by HarishGangula — APPROVED** [review]

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**